### PR TITLE
Some janitoring

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

File: src/opm/input/eclipse/Schedule/KeywordHandlers.cpp [hunk 1]
```diff
@@ -1532,6 +1532,7 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
                 if (well2.updateHasInjected())
                     update_well = true;
 
+                bool crossFlow = well2.getAllowCrossFlow();
```
Comment (bska) [inline src/opm/input/eclipse/Schedule/KeywordHandlers.cpp:1535]: Can you make this `const`, please?

File: src/opm/input/eclipse/Schedule/KeywordHandlers.cpp [hunk 2]
```diff
@@ -1614,6 +1615,7 @@ File {} line {}.)", wname, location.keyword, location.filename, location.lineno)
...
```
Comment (bska) [inline src/opm/input/eclipse/Schedule/KeywordHandlers.cpp:1618]: This too should be `const`.

**Review by bska — COMMENTED** [review]
Good catch!

File: src/opm/input/eclipse/Schedule/KeywordHandlers.cpp (hunk 1 above)
Comment (akva2) [reply src/opm/input/eclipse/Schedule/KeywordHandlers.cpp]: yeah, my bad.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me and I'll merge into master.

**Pull request merged**